### PR TITLE
Refactor bitesize5

### PR DESCRIPTION
**Pull request by t-young31**

### CODE REVIEW

File: tdms/src/iterator.cpp [hunk 1]
```diff
@@ -6150,47 +6149,6 @@ void initialiseDouble2DArray(double **inArray, int i_lim, int j_lim) {
     for (int i_var = 0; i_var < i_lim; i_var++) inArray[j_var][i_var] = 0.0;
 }
 
-//result gives field according to the exp(-iwt) convention
```
Comment (samcunliffe) [inline tdms/src/iterator.cpp:6153]: Should this comment be preserved somewhere?
(Maybe [line 57 of the new](https://github.com/t-young31/ifdtd/pull/7/commits/ce7b7fdec9a8a4abf476db7ec7b15d1f410100e7#diff-f613571e3523d1c68d7a9b71bc905e297c0bb6646ef8ac4c0602f1d2461696eeR57))

File: tdms/src/iterator.cpp [hunk 2]
```diff
@@ -6269,60 +6227,6 @@ void extractPhasorHNorm(complex<double> *Hnorm, double ft, int n, double omega,
...
-//
-//result gives field according to the exp(-iwt) convention
-void extractPhasorsVolumeH(MagneticField &H, MagneticSplitField &H_s,
-                           int n, double omega, double dt, int Nt) {
-
-  complex<double> phaseTerm, subResult;
-  double hx_m, hy_m, hz_m;
-
-  //a + 0.5 is added because we always know H half a time step
-  //after we know E.
```
Comment (samcunliffe) [inline tdms/src/iterator.cpp:6286]: And maybe add this to the [ElectricField::phase](https://github.com/t-young31/ifdtd/pull/7/commits/ce7b7fdec9a8a4abf476db7ec7b15d1f410100e7#diff-07de152f1fee2c5f2062108ebe267fb502b782270603d5d02037f8fba56f57dcR7) ?

(I don't insist)

File: tdms/matlabio/matlabio.cpp [hunk 3]
```diff
@@ -1,164 +1,13 @@
...
-	}
-	else{
-		double *verb = mxGetPr(prhs);
-		if( *verb > 0)
-			return 1;
-		else
-			return 0;
-	}
-}
-#endif
```
Comment (samcunliffe) [inline tdms/matlabio/matlabio.cpp:161]: I completely approve of deleting unused stuff... why is it now unused?
(Excuse the dumb question)

**Review by samcunliffe — APPROVED** [review]
This looks nice. Sorry for being so slow to catch up.

File: tdms/src/iterator.cpp (hunk 1 above)
Comment (t-young31) [reply tdms/src/iterator.cpp:6153]: it's preserved [here](https://github.com/t-young31/ifdtd/blob/ce7b7fdec9a8a4abf476db7ec7b15d1f410100e7/tdms/include/field.h#L179)

File: tdms/matlabio/matlabio.cpp (hunk 3 above)
Comment (t-young31) [reply tdms/matlabio/matlabio.cpp:161]: it was never used 👻

**Pull request merged**